### DISCUSSION TITLE
command console?

### DISCUSSION THREAD

**Discussion by zx3800** [winstxnhdw/lc-hax#253] [Ideas]
is there a way to create a console? i'm pretty sure that people have mods to read chat commands and all of that stuff

**Comment by winstxnhdw** [comment]
There is no mod that can ever read the commands you send. These commands are never sent to the internet in any form. The only way a mod would know whether you are typing a command would be if the mod is a keylogger malware, which would be illegal to distribute, and you would not be the smartest person to voluntarily install a keylogger.

**Reply by zx3800** [reply]
damn, the dudes must be super sleuths then to know im the one pulling the horn
will never know how they figure it out

**Reply by winstxnhdw** [reply]
It's not that hard when you see a person stop walking before every horn lol
